processor/src/main/java/org/mapstruct/ap/internal/model/MapMappingMethod.java

```diff
@@ -214,6 +221,10 @@ private Assignment forgeMapping(SourceRHS sourceRHS, Type sourceType, Type targe
             return assignment;
         }
 
+        private String stubPropertyName(String fullyQualifiedName) {
+            return Strings.decapitalize( fullyQualifiedName.substring( fullyQualifiedName.lastIndexOf( '.' ) + 1 ) );
```

Comment:
Hey, I know that one... If we really need it, the `Strings` class might actually a good place for it.